.github/workflows/fork-pr_local_integration_tests.yml

```diff
@@ -0,0 +1,66 @@
+name: fork-pr-local-integration-tests
```

Comment:
Is this meant to be used by forks? Can we place it in a different location as opposed to making it a valid workflow? That'll likely just pollute our actions page (https://github.com/feast-dev/feast/actions) even more than it already is. Maybe it can go into `.github/fork-workflows` or something?

Reply:
fixed